docs/extending_syzkaller.md

```diff
@@ -0,0 +1,39 @@
+# Extending syzkaller
+
+Here are the common files and some general edits that need to be made in order to extend syzkaller for a new OS kernel. However, there are some specific changes that may be required for a given kernel (For example gathering coverage from a given kernel, or some errors that might pop up and give a hint about what to tweak).
```

Comment:
comma after "For example"
and no capital letter in "For example" since it's not beginning of a sentence (or make it a separate sentence).